src/index.html

```diff
@@ -22,6 +22,508 @@
   </head>
 
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <a
+        class="header-logo"
+        href="№"
```

Comment:
The `href` attribute of the anchor tag is set to '№', which is not a valid URL. It should be a valid URL or a placeholder like '#'.